decidim-core/app/assets/javascripts/decidim/form_filter.component.js.es6

```diff
@@ -68,13 +71,44 @@
     }
 
     /**
+     * Parse current location and get filter values.
+     * @private
+     * @returns {Object} - An object where a key correspond to a filter field
+     *                     and the value is the current value for the filter.
+     */
+    _parseLocationFilterValues() {
+      // Every location param is constructed like this: filter[key]=value
+      let regexpResult = decodeURIComponent(this._getLocation()).match(/filter\[([^\]]*)\](?:\[\])?=([^&]*)/g);
```

Comment:
There is no native way to handle query params. You can find a lot of jQuery plugins or npm packages to do that but I think there is no need of that if you can use plain regular expressions. Thoughts?